projects/ngx-drag-to-select/src/lib/select-container.component.ts

```diff
@@ -168,6 +173,19 @@ export class SelectContainerComponent implements AfterViewInit, OnDestroy {
         share()
       );
 
+      const keydown$ = fromEvent<KeyboardEvent>(window, 'keydown');
+      const keyup$ = fromEvent<KeyboardEvent>(window, 'keyup');
+
+      this.selectBoxClasses$ = merge(dragging$, keydown$, keyup$).pipe(
+        auditTime(AUDIT_TIME),
+        map(event => {
+          return {
+            'dts-adding': !this.shortcuts.removeFromSelection(event) || !event,
```

Comment:
Why is `!event` needed here?

Reply:
Uhm, good catch. This is a left over. I ll remove it.